src/printer.js

```diff
@@ -1996,6 +2000,8 @@ function printFunctionParams(path, print, options) {
 
   return concat([
     "(",
+    comments.printDanglingComments(path, options, /* sameIndent */ true),
```

Comment:
No, but the reason was to be able to add the comments when no params are present and I can't add the comments on the params node (which is empty by the way).